test/examples/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
/bin/bash is not available in Alpine, I guess this is the reason for the failed CCI.

Reply:
It was part of it, the main reason was that `/bin/ash` (the default shell in Alpine), doesn't support double-quoted commands through `/bin/sh -c 'foo && bar'`, but we need double quotes to get the $ENGINE_IP variable value into the command :) It's sorted now.